### PR TITLE
feat: adds cofc process flowchart & report form

### PR DESCRIPTION
**Pull request by daffodil11**

### CODE REVIEW

**Comment by KeiranWDigital** [comment]
Please could you swap the branch you are merging to to be "release"

**Comment by daffodil11** [comment]
The Code of Conduct Committee has had an opportunity to review the changes to the Code of Conduct and this PR is now ready for general review.

File: code-of-conduct.md [hunk 1]
```diff
@@ -56,15 +56,13 @@ It may be appropriate for a recruitment company to sponsor refreshments and do a
 
 ### What to do if you have a concern
 
-If you are being harassed or feel uncomfortable, notice that someone else is being harassed, or have any other concerns, please speak to the event organiser(s) immediately. You can also contact the event organiser through Meetup and Slack.
+If you are being harassed or feel uncomfortable, notice that someone else is being harassed, or have any other concerns, CodeWith is here to listen to your concerns or questions. You don’t have to make a formal report unless you want to. You don't have to give us details of the harassment.
 
-If you are not able to to do this or you feel that your concern is not being taken seriously then please contact the Equity Officer. This is currently Marie Fayard, who can be reached on Discord, on Slack, or by email ([marie.fayard@gmail.com](mailto:marie.fayard@gmail.com)).
+If you wish to report something, please use [this form]({{site.url}}/code-of-conduct-report) to send a report directly to the Code of Conduct Committee.
```
Comment (KeiranWDigital) [inline code-of-conduct.md]: I'd prefer this to be something like "please use the Code Of Conduct Report Form" as its more accessible to those who use screen readers

**Review by KeiranWDigital — CHANGES_REQUESTED** [review]
Just one slight suggestion on the markdown file.

Also, I think we should update the text at the bottom to say when it was last updated. 

Thanks for putting this together, looks really good!

File: code-of-conduct.md (hunk 1 above)
Comment (daffodil11) [reply code-of-conduct.md]: Very good point. Updated.

**Comment by KeiranWDigital** [comment]
Happy with this, thanks :)

**Review by KeiranWDigital — APPROVED** [review]

**Pull request merged**